src/q2g-con-psexecute-qvx/App.config

```diff
@@ -1,6 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2"/>
-    </startup>
-</configuration>
+  <configSections>
+    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+  </configSections>
+  <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <targets>
+      <target xsi:type="File" name="file" keepFileOpen="true" openFileCacheTimeout="5" archiveNumbering="Date" autoFlush="true"
```

Comment:
@mossbeard please double check keepFileOpen from the docu, this sounds for me to set this better to false

Reply:
@konne I read the documentary for keepFileOpen and tested it with True under Server and Desktop. It works. I can also set it to false but the documentation says the performance was better with true.
https://github.com/NLog/NLog/wiki/Performance#file-logging-performance